src/qtcore/qml/elements/Item.js

```diff
@@ -84,6 +84,7 @@ function QMLItem(meta) {
     this.heightChanged.connect(this, updateVGeometry);
     this.implicitWidthChanged.connect(this, updateHGeometry);
     this.implicitHeightChanged.connect(this, updateVGeometry);
+    
```

Comment:
I'm working from web page :) (via github). That change is just to create a branch.